client/question/components/OutputSnippetDirective.js

```diff
@@ -20,55 +20,22 @@ tie.directive('outputSnippet', [function() {
   return {
     restrict: 'E',
     scope: {
-      getContent: '&content',
-      onStateChange: '&onStateChange'
+      getContent: '&content'
     },
     template: `
       <div class="output-toggle-container">
-        <a href class="toggle" ng-click="toggleOutput()">
-          {{outputIsShown ? 'Hide' : 'Display'}} output
-        </a>
-      </div>
-      <div class="output-container" ng-show="outputIsShown">
-        <span ng-repeat="line in snippetLines">
-          <span class="snippet-line">{{line}}</span>
-          <br>
-        </span>
+        You can click on <a href ng-click="openOutputModal()">this link</a>
```

Comment:
Yeah, fair point. Do you want it to not change in non-dark mode, too? If so, should it just stay as whatever the original colour of the link is?

Reply:
Yeah, I'm okay with it never having the 'visited' purpled link in either of the themes. Since it's likely that if you are using the app, you'll want to click through that link and it doesn't matter whether or not you've clicked it before?

Reply:
OK, sounds good -- done!